integration/airflow/openlineage/airflow/plugin.py

```diff
@@ -6,7 +8,12 @@
 # Provide empty plugin for older version
 from openlineage.airflow.macros import lineage_parent_id, lineage_run_id
 
-if parse_version(AIRFLOW_VERSION) < parse_version("2.3.0.dev0"):
+
+def _is_disabled():
+    return os.getenv("OPENLINEAGE_DISABLED", None) in [True, 'true', "True"]
```

Comment:
Since we know that extractors can have an impact on the runtime, can we add similar logic to the `LineageBackend` so that no code is executed when OPENLINEAGE_DISABLED is true?

Reply:
Done